pyproject.toml

```
@@ -0,0 +1,2 @@
+[tool.pytest.ini_options]
+asyncio_mode = "auto"
```

```suggestion
asyncio_mode = "strict"
```
This should be strict.